### PR TITLE
Premerge minor fixs

### PR DESCRIPTION
**Pull request by libbylondon11**

### CODE REVIEW

File: client/app/account/login/login.html [hunk 1]
```diff
@@ -6,7 +6,6 @@
     <div class="col-sm-12">
       <h1>Login</h1>
       <!--<p>Accounts are reset on server restart from <code>server/config/seed.js</code>.--> Test account is <code>test@test.com</code> / <code>test</code></p>
-      <!--<p>Admin account is <code>admin@admin.com</code> / <code>admin</code></p>-->
```
Comment (gemfarmer) [inline client/app/account/login/login.html:9]: keep

File: client/app/main/main.scss [hunk 2]
```diff
@@ -79,18 +79,38 @@
 }
 
 .community-image {
-    width: 90%;
+    width: 38%;
     float: left;
-    margin-right: $base-padding;
+    margin-right: 5%;
```
Comment (gemfarmer) [inline client/app/main/main.scss]: this should probably stay `$base-padding`

File: client/app/main/warmupModal.html [hunk 3]
```diff
@@ -1,10 +1,40 @@
-<!--<div>Pre-dream warm-up text here.</div>-->
-<img class="community-image" src="assets/images/community.png"/>
-<!--<div>Dream Brainstorm
-Take 5-8 minutes to put your dreams and goals on paper.  Feel free to brainstorm with lists, drawings, or figures etc., whatever you need to get the ideas flowing. This will be yours to keep and private to you. Your dreams may look very different today then tomorrow and that is okay. You’re just trying to get the ideas flowing and on paper.
+<!DOCTYPE html>
```
Comment (gemfarmer) [inline client/app/main/warmupModal.html:1]: <html> shouldn't be inside another <html> tag (with the exception of iframes

File: client/app/main/main.scss [hunk 4]
```diff
@@ -79,18 +79,38 @@
...
     margin-left: $base-padding;
 }
-.brainstorm-image {
-    width: 90%;
-    float: left;
+
+.world-image {
```
Comment (libbylondon11) [inline client/app/main/main.scss:88]: uses a different image "world.png" - approve

File: client/app/main/main.scss [hunk 5]
```diff
@@ -79,18 +79,38 @@
...
+    width: 27%;
+    float: right;
     margin-right: $base-padding;
     margin-left: $base-padding;
+    position: relative;
+    top: 1px;
 }
+
 .modal {
```
Comment (libbylondon11) [inline client/app/main/main.scss]: changes size of the modal?  I'm not sure where this class was before

File: client/app/main/warmupModal.html [hunk 6]
```diff
@@ -1,10 +1,40 @@
...
+<html>
+    <head>
+        <title>Predream_warmup</title>
+</head>
+
+<body id="textbody"> 
+  <div class="text-padding">   
+ 	<h3 id="title2"> Affirm and Inspire </h3>
+	<p>• Take a moment to put yourself in a place of acceptance, a place of positivity where anything is possible.</p>
```
Comment (libbylondon11) [inline client/app/main/warmupModal.html]: could add ul li inside p tags

File: client/app/routes/resources/resources.html [hunk 7]
```diff
@@ -1,12 +1,46 @@
 <div ng-include="'components/navbar/navbar.html'"></div>
 <div ng-include="'components/sideNav/sideNav.html'" ></div>
 
-<div class="page">
+<base target="_blank">
+
+<div class="page"></div>
```
Comment (libbylondon11) [inline client/app/routes/resources/resources.html]: I think close the page class div at the end of the page

**Pull request merged**